iron-scroll-target-behavior.html

```diff
@@ -189,15 +189,21 @@
      * Scrolls the content to a particular place.
      *
      * @method scroll
-     * @param {number} left The left position
-     * @param {number} top The top position
+     * @param {number|!ScrollToOptions} leftOrOptions The left position or scroll options
+     * @param {number=} top The top position
+     * @return {void}
      */
-    scroll: function(left, top) {
-       if (this.scrollTarget === this._doc) {
-        window.scrollTo(left, top);
+    scroll: function(leftOrOptions, top) {
+      var opts = leftOrOptions;
+      if (typeof leftOrOptions !== 'object') {
```

Comment:
Maybe unpack the options into a left/top var if present, instead of always allocating this object.